### ISSUE TITLE
Edit-site package: Latest package versions contain outdated file

### ISSUE THREAD

**Issue by ockham** [WordPress/gutenberg#24941]
**Describe the bug**
I'm trying to use [`@wordpress/edit-site`'s](https://www.npmjs.com/package/@wordpress/edit-site) current version (1.11.7, published 5 days ago ) in a project (https://github.com/Automattic/wp-calypso/pull/44928). However, I've noticed that the package's `src/index.js` file doesn't reflect the [current state](https://github.com/WordPress/gutenberg/blob/379719fe72699b0e2077d8263f5d78a0e3eecd33/packages/edit-site/src/index.js) but [a state prior to the last PR](https://github.com/WordPress/gutenberg/pull/23775/files#diff-ce213e27f12503824dc8b2066c47a7b6) that touched that file (#23775, merged on July 20 -- more than a month ago).

I'm somewhat concerned that our package release script might be broken, and I wonder if this also affects other packages.

**To reproduce**
1. Clone the Automattic/wp-calypso repo.
2. Check out the branch attached to https://github.com/Automattic/wp-calypso/pull/44928.
3. Run `yarn` (or `yarn distclean && yarn`, if you already had it checked out).
4. Inspect `node_modules/@wordpress/edit-site/src/index.js`. You'll notice that it has this part in `initialize` (which should no longer be there, as of #23775):

```
	const initialState = settings.editSiteInitialState;
	delete settings.editSiteInitialState;
	initialState.settings = settings;
```

**Expected behavior**
The `src/index.js` file in the `@wordpress/edit-site` package to reflect the [current state](https://github.com/WordPress/gutenberg/blob/379719fe72699b0e2077d8263f5d78a0e3eecd33/packages/edit-site/src/index.js).

**Additional context**
The problem was also present in the previous version of the package, 1.11.6, which was also published after #23775 was merged.

cc/ @gziolo @youknowriad @mcsf @noahtallen @vindl @sirreal

**Comment by youknowriad** [comment]
There's no problem here, I don't think we many any major npm release since WordPress 5.5 beta 1. We could start doing these again.

**Comment by ockham** [comment]
> There's no problem here, I don't think we many any major npm release since WordPress 5.5 beta 1. We could start doing these again.

Not sure I understand -- does that mean that minor package releases contain the exact same snapshots of files as the latest major one? :thinking:

**Comment by youknowriad** [comment]
no, it means we made some package releases recently with some specific cherry-picks into the `wp/5.5` branch and we didn't include the latest changes from the Gutenberg plugin/master.

We do pause the npm releases (major ones) during the beta/RC cycle of WordPress releases but now we can resume doing so after plugin releases.

**Comment by youknowriad** [comment]
https://github.com/WordPress/gutenberg/blob/master/docs/contributors/release.md#synchronizing-wordpress-trunk this is what I call "major npm releases"

**Comment by ockham** [comment]
Okay, in that case, it'd be great if we could release packages.

Still trying to wrap my head around what the implications are for minor package releases (i.e. what kind of changes _were_ being published).

**Comment by youknowriad** [comment]
> Still trying to wrap my head around what the implications are for minor package releases (i.e. what kind of changes were being published).

Bug fixes meant to be backported to WordPress 5.5 and 5.5.1. Basically all PRs that had the "Backport to WP" label.

**Comment by ockham** [comment]
> > Still trying to wrap my head around what the implications are for minor package releases (i.e. what kind of changes were being published).
> 
> Bug fixes meant to be backported to WordPress 5.5 and 5.5.1. Basically all PRs that had the "Backport to WP" label.

Gotcha, thanks! :+1:

**Comment by gziolo** [comment]
It looks like it isn’t actionable.
